applications/solana_vm_on_polkadot_sdk.md

```diff
@@ -0,0 +1,127 @@
+
+## Team
+
+- **Team Name:** Haderech
+- **Contact Name:** Jeeyong Um
+- **Contact Email:** conr2d@haderech.com
+- **Website:** https://github.com/haderech
+
+### Team members
+
+- Jeeyong Um
+- Jungyong Um
+
+#### LinkedIn Profiles
+
+- https://www.linkedin.com/in/jeeyong-um
+- https://www.linkedin.com/in/jungyong-um
+
+### Team Code Repos
+
+- https://github.com/noirhq/noir
+- https://github.com/noirhq/solana-sdk
+
+#### GitHub Profiles
+
+- https://github.com/conr2d
+- https://github.com/code0xff
+
+### Team's experience
+
+- Extensive open-source contributions (Polkadot SDK, Frontier, Cosmos SDK, EOS, etc.)
+- Alumni of Polkadot Blockchain Academy (PBA)
+  - First prize winner at PBA Demo Day
+
+## Development Status
+
+Proof of Concept implementations:
+
+- `pallet-solana` PoC [[link]](https://github.com/NoirHQ/noir/tree/c681ad696586a686b5af1a2afab32d2f1a52acaa/frame/solana)
+- `solana-sdk` with `no_std` support [[link]](https://github.com/NoirHQ/solana-sdk/tree/4ea2ba03098f440b934985a31ab00950387331b9/sdk)
+- `solana_rbpf` with `no_std` support [[link]](https://github.com/NoirHQ/solana-sdk/tree/4ea2ba03098f440b934985a31ab00950387331b9/rbpf)
+
+## Development Roadmap
+
+### Overview
+
+- **Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):** 2.5 FTE
+- **Total Costs:** 30,000 USD
+
+> Note that deliverables 0a to 0e are mandatory. Please adapt their specification to your project.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | pallet-solana (GPLv3), sidecar (GPLv3), template-runtime (Unlicense) |
+| 0b. | Documentation | Inline documentation of the code and a basic tutorial on integrating `pallet-solana` into Polkadot SDK runtimes |
+| 0c. | Testing and Testing Guide | Complete unit and integration tests for vaildation |
+| 0d. | Docker | Dockerfile for testing functionality |
+| 0e. | Article | Publish an article on the Polkadot forum and public blogging platforms |
+| 1. | Pallet Solana | Develop `pallet-solana` to process Solana transactions within Polkadot SDK runtimes |
+| 2. | Publishing | Publish packages on `crates.io`, where feasible |
+| 3. | Sidecar | Create Solana RPC emulation for wallet interaction |
+| 4. | Template Runtime | Provide a template runtime with `pallet-solana` for testing and development |
+| 5. | Minimal Testing UI | Develop a minimal web UI for testing wallet interactions |
+
+### Budget Breakdown
+
+| Category | Item | Cost | Amount | Total | Description |
+| --- | ---- | --- | --- | --- | ---|
+| Personnel | Blockchain Architect | 15,000 USD | 1.0 FTE | 15,000 USD | Architecture design and development |
+| Personnel | Blockchain Developer | 10,000 USD | 1.5 FTE | 15,000 USD | Development, testing and documentation |
+| --- | --- | --- | **Total** | **30,000 USD** |  |
+
+## Future Plans
+
+The outcomes of this project will be utilized in Haderech's ongoing projects, and be maintained and developed alongside the Polkadot SDK.
```

Comment:
Could you elaborate what these ongoing projects are?

Reply:
I am building a non-profit project named by [Noir](https://github.com/noirhq/noir), an extension to Polkadot SDK for building a multi-vm chains. This aims to solve diversified protocol specification issues by integrating EVM, WASM, SVM (Solana VM) and MOVE VM, so that any wallet users, any dapps can connect to Polkadot ecosystem seamlessly.